package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-topbar-extensions",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Do we need to bump this manually?

Reply:
The check-release job fails otherwise: https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/114

Reply:
But we should handle that better (as part of https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/114)

Reply:
Agree!! We need to improve this. Do you know if there are any features added in `jupyter-releaser` to handle monorepos? I will take a look into it as well.

Reply:
Monorepos of JS packages should work fine. However it's a bit more tricky with Python monorepos.

There are some issues on the releaser repo somehow related:
- https://github.com/jupyter-server/jupyter_releaser/issues/496
- https://github.com/jupyter-server/jupyter_releaser/issues/499

This would also be useful for releasing `ipywidgets` with the releaser: https://github.com/jupyter-widgets/ipywidgets/issues/3285

Reply:
Cheers for the links. I think a custom script to handle Python packages as you suggested in [jupyter_releaser#496](https://github.com/jupyter-server/jupyter_releaser/issues/496) can be viable option. 

I can look into it in the next weeks!

Reply:
Sounds good, thanks!